IsotropicTextureMat.h

```diff
@@ -69,8 +69,14 @@ class IsotropicTextureMat : public LinIsotropic
                 const FiniteElement& fe, const Vec3& X) const override;
 
 protected:
+  struct rgba {
+    double r;
+    double g;
+    double b;
+    double a;
+  };
   std::map<std::pair<double,double>,LinIsotropic> materials; //!< Material for different texture regions. 
-  std::string textureFile; //!< Path to texture file
+  std::vector<std::vector<struct rgba> > textureData;
```

Comment:
How about `std::vector< std::vector< std::array<double,4> > >` instead, since you want it as an array in the parse-method anyway. And the doxy is missing for this member as well.

Reply:
Both are good options. @akva2 suggested `struct rgba` (which I feel is kind of self-documenting), and you suggest `std::array<double,4>`. I guess @kmokstad outranks you in this particular repository akva2 so I'll go with the latter option.

Reply:
I don't insist on using the array instead of a struct, but it generates doxy-warnings during `make doc` if not documented, no-matter how self-explanatory it might be.